securemailbox/views/retrieve.py

```diff
@@ -1,11 +1,69 @@
-from flask import Blueprint, jsonify
+from flask import Blueprint, jsonify, request, json
+
+from ..models import Mailbox
+from ..models import Message
 
 from securemailbox import db
 
 # Create the `retrieve` blueprint
 retrieve_blueprint = Blueprint("retrieve", __name__)
 
-
 @retrieve_blueprint.route("/retrieve/", methods=["POST"])
 def retrieve():
-    return jsonify({"success": True, "error": None})
+
+    data = request.get_json(force=True, silent=True)
```

Comment:
After look at the [docs](https://flask.palletsprojects.com/en/1.1.x/api/#flask.Response.get_json) for `get_json` I have a nit:

I would recommend that instead of intentionally suppressing JSON decode errors to None, you instead explicitly check that the request is encoded as `application/json`. An example of this behavior can be found [here](https://github.com/securemailbox/api/blob/bcef9f20673e24f5106b2414f042b1ea3942e832/securemailbox/views/register.py#L16). The reason for this being that I think it's clearer to someone unfamiliar with flask and this project in general. Also, we've explicitly stated that the API will exclusively speak JSON.